docs/using.md

```diff
@@ -65,6 +66,33 @@ its runtime state into _~/.minishift_. Using _MINISHIFT_HOME_, you can choose a
 as Minishift's home directory. This is currently experimental and semantics might change in
 future releases.
 
+<a name="config-file"></a>
+## Config file
+
+Minishift also maintains a config file (_~/.minishift/config/config.json_) which can be used to set some
+variables like (CPU, memory ...etc.) and can be used without using different parameter in start command.
+This can be used to control some of the default behavior similar to using
+[environment variables](#environment-variables).
+
+**Note:**
+
+* config.json file is not generated automatic until user use `set` sub-command as described below.
+* Manual edit to this file is discouraged because it might be error-prone, use defined sub-command for required use-case.
+
+```shell
+$ minishift config -h
+```
+
```

Comment:
We should reference to the generated help page for the config command, since there the available properties are listed. Otherwise we need to list it somewhere in here as well